### PR TITLE
PERF-148 JMeter script to test POST loan-storage performance

### PR DESCRIPTION
**Pull request by javalkar-varun**
https://issues.folio.org/browse/PERF-148 Test POST /loan-storage/loans

**Approach:**
- Make a direct call to POST /loan-storage/loans to create loans in bulk. Around 72k loans are created in 30 minutes.

### CODE REVIEW

File: workflows-scripts/circulation/loan-storage/circulationStorage_POSTLoansStorage.jmx [hunk 1]
```diff
@@ -0,0 +1,375 @@
...
+          <stringProp name="filename">${global_BaseDir}/jmeter-supported-data/itemIds90000.csv</stringProp>
+          <boolProp name="ignoreFirstLine">false</boolProp>
+          <boolProp name="quotedData">false</boolProp>
+          <boolProp name="recycle">true</boolProp>
+          <stringProp name="shareMode">shareMode.all</stringProp>
+          <boolProp name="stopThread">false</boolProp>
+          <stringProp name="variableNames">itemId</stringProp>
+        </CSVDataSet>
+        <hashTree/>
+        <TransactionController guiclass="TransactionControllerGui" testclass="TransactionController" testname="Create Loans Transaction" enabled="true">
```
Comment (mtraneis) [inline workflows-scripts/circulation/loan-storage/circulationStorage_POSTLoansStorage.jmx:229]: Just curious, why is there a need for the Transaction controller here?

**Review by mtraneis — APPROVED** [review]
Looks great for the most part.
I think this is an API test and it should be moved to /api-scripts, do you agree?

**Comment by javalkar-varun** [comment]
> I think this is an API test and it should be moved to /api-scripts, do you agree?
Sounds good. I will move to /api-scripts directory.

**Comment by javalkar-varun** [comment]
Accidentally closed the PR but reopened it again.
> I think this is an API test and it should be moved to /api-scripts, do you agree?


Sounds good. I will move to /api-scripts directory.

File: workflows-scripts/circulation/loan-storage/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+In this workflow, we are checking the performance of POST /loan-storage/loans API for creating loans in bulk. IC's normally create loans in bulk using mod-circulation business logic which is slow. This script will investigate if the same can be done using the mod-circulation-storage module bypassing business logic.
+
+### Pre-requisite:
+#### Generate data:
+1. Log into database
+2. Run Postgresql query to generate item ids 
+For example ```	SELECT distinct id FROM fs09000000_mod_inventory_storage.item where jsonb->'status'->>'name'='Available' LIMIT 90000;```
+3. Save all ids in itemsId.csv file and use it in JMeter for creating loan
+
```
[the remark this answers is not in the document]
Comment (javalkar-varun) [reply workflows-scripts/circulation/loan-storage/README.md]: Updated readme

File: workflows-scripts/circulation/loan-storage/circulationStorage_POSTLoansStorage.jmx (hunk 1 above)
Comment (javalkar-varun) [reply workflows-scripts/circulation/loan-storage/circulationStorage_POSTLoansStorage.jmx]: You are right, Transaction controller is not required. There is only one HTTP POST request. I have removed it.

**Pull request merged**